enclave-runtime/src/lib.rs

```diff
@@ -569,7 +563,7 @@ where
 
 	let xts = extrinsics_factory.create_extrinsics(calls.as_slice())?;
 
-	validator.send_extrinsics(on_chain_ocall_api, xts)?;
+	validator_access.execute_mut_on_validator(|v| v.send_extrinsics(on_chain_ocall_api, xts))?;
```

Comment:
Could you add a short comment here why mut access is needed? I myself (and I know the code a little by now) stumbled upon that a little.

`// Access needs to be mutable to store sent extrinsics in inclusion_check buffer`.

or something like that.